client/components/domains/map-domain-step/index.jsx

```diff
@@ -59,8 +60,7 @@ const MapDomainStep = React.createClass( {
 
 	render: function() {
 		const suggestion = { cost: this.props.products.domain_map.cost_display, product_slug: this.props.products.domain_map.product_slug },
```

Comment:
I believe the `domain_map` can't be `undefined`? Otherwise, we'd need the same condition from below.